### ISSUE TITLE
Strange bugs occur when the number of Gpus trained and tested is inconsistent

### ISSUE THREAD

**Issue by Freedom-JJ** [victoresque/pytorch-template#104]
If the number of Gpus is inconsistent, the number of files loaded by the loader does not correspond to BatchSize, and the prediction accuracy is always 0.166666
